# KeyData.msg
string router_key
string display_key

